make.js

```diff
@@ -307,11 +309,22 @@ target.dist = function() {
     GENERIC_DIR + 'build/pdf.js',
     GENERIC_DIR + 'build/pdf.worker.js',
     SINGLE_FILE_DIR + 'build/pdf.combined.js',
+    MINIFIED_DIR + 'build/pdf.min.js',
+    MINIFIED_DIR + 'build/pdf.worker.min.js',
+    MINIFIED_DIR + 'build/pdf.combined.min.js'
   ], DIST_DIR + 'build/');
 
   mkdir('-p', DIST_DIR + 'web/');
   cp('-R', [
     COMPONENTS_DIR + '*',
+    GENERIC_DIR + 'web/viewer.js',
```

Comment:
I think you're right, it's maybe not in the scope of this PR, but I also think it would be good to publish a minified viewer. 

The use case I have in mind is if we want to export a generic viewer that can be included "out of the box" in other projects. Perhaps this is not the scope of the pdf.js project (it could be extracted into its own project that uses pdf.js), but then perhaps the whole viewer should not be included in this repo in that case. I appreciate it's quite a big and perhaps unnecessary change but if we're saying that the viewer shouldn't be published then it might make sense to put the viewer into its own repo and not version it here. 

I guess my question is: what is the role of the viewer in this repo? If it is a product that is bring released then maybe there's no harm on releasing a minified version.